lib/Models/addUserCatalogMember.ts

```diff
@@ -45,31 +46,36 @@ export default function addUserCatalogMember(
       );
       terria.catalog.userAddedDataGroup.add(CommonStrata.user, newCatalogItem);
 
+      const dereferenced =
```

Comment:
Maybe it's time we port over the helper I have in the editor to terriajs#mobx
https://github.com/TerriaJS/terrace/blob/995a33cfc588a7d7d7a4cc3f607982dc02d3d083/lib/Helpers/referenceHelpers.ts#L10